docs/Migrating.md

```diff
@@ -1,5 +1,34 @@
 # Migrating
 
+## Migrating from ML-Agents toolkit v0.4 to v0.5
+
+### Important
+
+* The Unity project `unity-environment` has been renamed `MLAgentsSDK`.
+* The `python` folder has been renamed to `ml-agents`. It not contains two
+  packages, `mlagents.env` and `mlagents.trainers`. `mlagents.env` can be used
+  to interact directly with a Unity environment, while `mlagents.trainers`
+  contains the classes for training agents.
+
+### Unity API
+
+* Discrete Actions now have branches. You can now specify concurrent discrete
+  actions. You will need to update the Brain Parameters in the Brain Inspector
+  in all your environments.
+
+### Python API
+
+* In order to run a training session, you must now use the command
+  `mlagents-learn` instead of `python3 learn.py` after installing the `mlagents`
+  packages.
+* It is now required to specify the path to the yaml trainer configuration file
+  when running `mlagents-learn`. For example, see
+  [trainer_config.yaml](../config/trainer_config.yaml).
```

Comment:
Why did we make this change? Do we have an example of the previous command vs current command?

Reply:
The reasoning for this change was that no matter what environment we're training, we need a trainer configuration file. Although we have sensible defaults for the environments we ship in `trainer_config.yaml`, it doesn't makes sense to have users add their custom configurations to this file or potentially get confused when default configurations are getting loaded.

By forcing the user to specify the configuration file, it makes it explicit where the hyper-parameters are coming from.